SampleApps/SPMTest/SPMTest.xcodeproj/project.pbxproj

```diff
@@ -449,81 +443,71 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
```

Comment:
This Build post-action workaround for XCode 12.4 can also be removed from the SPMTest App.